lib/explorer/data_frame.ex

```diff
@@ -664,13 +664,21 @@ defmodule Explorer.DataFrame do
   @spec to_csv(df :: DataFrame.t(), filename :: fs_entry() | String.t(), opts :: Keyword.t()) ::
           :ok | {:error, Exception.t()}
   def to_csv(df, filename, opts \\ []) do
-    opts = Keyword.validate!(opts, header: true, delimiter: ",", streaming: true, config: nil)
+    opts =
+      Keyword.validate!(opts,
+        header: true,
+        delimiter: ",",
+        quote_style: :necessary,
```

Comment:
We need documentation on the supported values for this function and we are good to go :)

Reply:
Done, let me know if there's any additional places needed